### DISCUSSION TITLE
Where is milkman data stored?

### DISCUSSION THREAD

**Discussion by strilling-hbs** [warmuuh/milkman#203] [General]
I was wondering where milkman data is stored? While using the Http Request Plugin, I deleted some of my requests. I was hoping to find where this data is stored and restore it from a back up. 

Thanks!

**Comment by warmuuh** [comment]
discussed offline, but for the record:

generally, the data is stored either in the installation directory of milkman, of, if not writable, in one of the directories:
Win: %LOCALAPPDATA%/Milkman/...
Mac: ~/Library/Application Support/Milkman/...
Linux: ~/.milkman/...
